charts/redpanda/chart.go

```diff
@@ -45,6 +52,31 @@ var (
 	Chart = gotohelm.MustLoad(chartYAML, defaultValuesYAML, render, console.Chart, connectors.Chart)
 )
 
+// Types returns a slice containing the set of all [kube.Object] types that
+// could be returned by the redpanda chart.
+func Types() []kube.Object {
```

Comment:
Looking at the output of the linter you might consider tag that would exclude this function from transpilation.

https://github.com/redpanda-data/helm-charts/actions/runs/11559675098/job/32174854859#step:5:337